tools/bazar/presentation/templates/agenda.tpl.html

```diff
@@ -111,4 +115,4 @@ <h5><?php echo $fiche['bf_titre'];?></h5>
 </style>
 
 <?php endif; ?>
-<?php echo $pager_links;?>
\ No newline at end of file
+<?php echo $pager_links;?>
```

Comment:
a mon avis il faut virer celui là (t'en a 2 dans le template : un avec des div autour et celui là)

Reply:
si on a besoin d'un div autour de $pager_link (aucune idée perso), il faudrait juste faire un test avant si $pager_links existe afin de ne pas avoir un <div></div> pour le cas où on n'a pas de pager.

Reply:
yes, a mon avis on peut meme virer le div, car le pager vient avec toutes les classes bootstrap qui vont bien

Reply:
Hum, 
En fait j'ai fais à partir d'une version mise à jour avec le bouton "mettre à jour" de mon wiki de test, alors que ça avait été déjà corrigé dans la source git (on trouve bien la ligne en doublon à la toute fin)... 
Donc note à moi même : la prochaine fois je vérifie que je pars bien de la version à jour sur le dépot -_- 
C'était un cas pratique d'entraînement pour m'assurer que je serai capable de faire une proposition de modif une fois seul devant mon écran, a priori du coup la prochaine fois ça ira mieux. Désoler pour le temps perdu :/  
MErci pour l'apprentissage !

Reply:
welcome in the dev world ! :man_technologist:

Reply:
pour info, c'est @furax37 qui a ajouté le code des pagination il y a 6 jours.  
Vous aviez encore bu trop de pastis.. ;)